app/src/main/java/org/sopt/at/presentation/home/HomeScreen.kt

```diff
@@ -0,0 +1,144 @@
+import org.sopt.at.R
+import org.sopt.at.presentation.home.component.BannerCarousel
+import org.sopt.at.presentation.home.component.HomeTopBar
+import org.sopt.at.presentation.home.component.RecommendContent
+import org.sopt.at.presentation.home.model.TabGenreResource
+import org.sopt.at.ui.theme.ATSOPTANDROIDTheme
+import org.sopt.at.ui.theme.Background
+import org.sopt.at.ui.theme.ButtonDisableText
+import org.sopt.at.ui.theme.White
+
+@Serializable
+data object Home
+
+@OptIn(ExperimentalFoundationApi::class)
+@Composable
+fun HomeScreen(
+    paddingValues: PaddingValues,
+    viewModel: HomeViewModel = viewModel(),
+    userId: String,
+    navigationToMy: (id: String) -> Unit
+) {
+    val tabTitles = stringArrayResource(R.array.home_tab_array)
+    var selectedTabIndex = viewModel.selectedTabIndex.collectAsState()
+    Column(
+        modifier = Modifier
+            .padding(paddingValues)
+            .fillMaxSize(),
+        verticalArrangement = Arrangement.Top,
+        horizontalAlignment = Alignment.CenterHorizontally,
+    ) {
+        HomeTopBar(
+            modifier = Modifier,
+            onProfileClick = { navigationToMy(userId) }
+        )
+        LazyColumn(
+            verticalArrangement = Arrangement.spacedBy(20.dp),
+            contentPadding = PaddingValues(vertical = 8.dp),
+            modifier = Modifier
+                .fillMaxSize()
+        ) {
+            stickyHeader {
+                HomeScrollableTabRow(
+                    tabTitles = tabTitles.toList(),
+                    selectedTabIndex = selectedTabIndex.value,
+                ) { tabIndex ->
+                    viewModel.selectTab(tabIndex)
+                }
+            }
+            item {
+                BannerCarousel(
+                    bannerImageUrls = TabGenreResource.getDataById(selectedTabIndex.value)!!.bannerImages,
+                )
+            }
+            item {
+                RecommendContent(
+                    modifier = Modifier,
+                    title = stringResource(R.string.home_recommend_live_popular, tabTitles[selectedTabIndex.value]),
+                    isSupportRanking = true,
+                    imageUrls = TabGenreResource.getDataById(selectedTabIndex.value)!!.posterImages,
+                    isShowMoreButton = false
+                )
+            }
+            item {
+                RecommendContent(
+                    modifier = Modifier,
+                    title = stringResource(R.string.home_recommend_current_broadcast),
+                    isSupportRanking = false,
+                    isShowMoreButton = true,
+                    imageUrls = TabGenreResource.getDataById(selectedTabIndex.value)!!.posterImages.reversed()
+                )
+            }
+        }
+    }
+}
+
+@Composable
+fun HomeScrollableTabRow(
+    modifier: Modifier = Modifier,
+    tabTitles: List<String>,
+    selectedTabIndex: Int,
+    onTabClick: (Int) -> Unit
```

Comment:
저도 이번 솝트하면서 알았는데 컴포저블 인자 순서는
필수, modifier, 선택 순서라고 합니다.
```suggestion
@Composable
fun HomeScrollableTabRow(
    tabTitles: List<String>,
    selectedTabIndex: Int,
    onTabClick: (Int) -> Unit
    modifier: Modifier = Modifier,
    text: String = "예시 텍스트입니다"
```

요런 순서로 넣는게 컴포즈 컨벤션이라고 하더라고요ㅎㅎ